src/builders/SquareRequestBuilder.php

```diff
@@ -59,6 +59,7 @@ public function buildChargeRequest(array $prepData)
         $request->setAutocomplete($prepData['autocomplete']);
         $request->setLocationId($prepData['location_id']);
         $request->setNote($prepData['note']);
+        $request->setVerificationToken($prepData['verification_token']);
```

Comment:
I would prefer if we apply the same approach here for the request, only set the attribute if it's value exists:

```php
if (array_key_exists('verification_token', $prepData)) $request->setVerificationToken($prepData['verification_token']);
```